CHANGELOG.md

```diff
@@ -14,8 +14,9 @@
 
 * **[Fix]** Fix an issue where invalid characters in the document ID are accepted at creation time but causing errors while trying to read or delete the document. The characters are `#`, `\`, `/`, `?`, and all whitespaces.
 
-### AppCenterDistribute
+### App Center Distribute
```

Comment:
We didn't use spaces in the previous versions. I guess we should harmonize it

Reply:
We just don't update it retro-actively for now unless someone feels like doing that.